kubeflow/fairing/deployers/job/job.py

```diff
@@ -44,7 +51,11 @@ def __init__(self, namespace=None, runs=1, output=None,
         self.deployment_spec = None
         self.runs = runs
         self.output = output
-        self.backend = KubeManager()
+        self.backend = KubeManager(
+            config_file=config_file,
+            context=context,
+            client_configuration=client_configuration,
+            persist_config=persist_config)
```

Comment:
Overall LGTM.

I feel that these four attributes (`config_file , context , client_configuration, persist_config `) related to config can be packed in structure in base class rather passing them individually everywhere in `pytorchjob, tfjob, kfserving` etc

Reply:
Thanks @gautamkmr For PythonJob and TFJob, they are have same basic class so I pass the args to super, but for kfserving, that's not sharing with basic class. And TFJob, PytorchJob and KFServing are from 3 different packages, I think that's hard to set in one place.